crates/web-sys/webidls/enabled/CanvasRenderingContext2D.webidl

```diff
@@ -107,33 +107,31 @@ interface CanvasRenderingContext2D {
   void demote();
 };
 
-CanvasRenderingContext2D implements CanvasState;
-CanvasRenderingContext2D implements CanvasTransform;
-CanvasRenderingContext2D implements CanvasCompositing;
-CanvasRenderingContext2D implements CanvasImageSmoothing;
-CanvasRenderingContext2D implements CanvasFillStrokeStyles;
-CanvasRenderingContext2D implements CanvasShadowStyles;
-CanvasRenderingContext2D implements CanvasFilters;
-CanvasRenderingContext2D implements CanvasRect;
-CanvasRenderingContext2D implements CanvasDrawPath;
-CanvasRenderingContext2D implements CanvasUserInterface;
-CanvasRenderingContext2D implements CanvasText;
-CanvasRenderingContext2D implements CanvasDrawImage;
-CanvasRenderingContext2D implements CanvasImageData;
-CanvasRenderingContext2D implements CanvasPathDrawingStyles;
-CanvasRenderingContext2D implements CanvasTextDrawingStyles;
-CanvasRenderingContext2D implements CanvasPathMethods;
-CanvasRenderingContext2D implements CanvasHitRegions;
```

Comment:
Could we interpret `A implements B` as `A includes B` in webidl parsing? I actually have no idea what the difference is between the two...

Reply:
We could but (a) it isn't standard webidl, and (b) there isn't any of that anymore after this PR :-p

Reply:
Heh ok!